docs/specs/localization.yml

```diff
@@ -373,10 +394,6 @@ outputs:
   xrefmap.json:
```

Comment:
specify xrefmap.json content as well?

Reply:
good idea

---
In reply to: [231406503](https://github.com/dotnet/docfx/pull/3585#discussion_r231406503) [](ancestors = 231406503)

Reply:
the below test has already covered, will remove it

---
In reply to: [231424602](https://github.com/dotnet/docfx/pull/3585#discussion_r231424602) [](ancestors = 231424602,231406503)